content/tutorials/build-apps/build-a-browser-wallet-in-js.md

```diff
@@ -0,0 +1,194 @@
+
+![Home Page Screenshot](img/js-wallet-home.png)
+
+This application can:
+
+- Show updates to the XRP Ledger in real-time.
+- View any XRP Ledger account's activity, including showing how much XRP was delivered by each transaction.
+- Show how much XRP is set aside for the account's [reserve requirement](reserves.html). 
+- Send [direct XRP payments](direct-xrp-payments.html), and provide feedback about the intended destination address, including:
+    - Displaying your account's available balance
+    - Verifying that the destination address is valid
+    - Validating the account has enough XRP to send
+    - Allowing you to specify a destination tag
+
+## Steps
+
+Before you begin, make sure you have the prerequisites installed. Check your node version by running `node -v`. If necessary, [download Node.js](https://nodejs.org/en/download/).
+
+**Tip:** If you get stuck while doing this tutorial, or working on another project, feel free to ask for help in the XRPL's [Developer Discord](https://discord.com/invite/KTNmhJDXqa).
+
+### 1. Set up the project
+
+1. Navigate to the directory that you want to create the project in.
+2. Create a new Vite project:
+
+```bash
+yarn create vite simple-xrpl-wallet --template vanilla
+```
+
+3. Create or modify the file `package.json` to have the following contents:
+
+{{ include_code("_code-samples/build-a-wallet/js/package.json", language="js") }}
+
+   - Alternatively you can also do `yarn add <package-name>` for each individual package to add them to your `package.json` file.
+  
+4. Install dependencies:
+
+```bash
+yarn
+```
+
+5. Create a new file `.env` in the root directory of the project and add the following variables:
+
+```bash
+CLIENT="wss://s.altnet.rippletest.net/" // or any other server
+EXPLORER_NETWORK="testnet" // or "mainnet"
+SEED="s████████████████████████████"  // Replace with your seed
+```
+
+6. Change the seed to your own seed. You can get credentials from [the Testnet faucet](xrp-test-net-faucet.html).
+
+7. Set up a Vite bundler. Create a file named `vite.config.js` in the root directory of the project and fill it with the following code:
+
+{{ include_code("_code-samples/build-a-wallet/js/vite.config.js", language="js") }}
+
+This example includes the necessary configuration to make [xrpl.js work with Vite](https://github.com/XRPLF/xrpl.js/blob/main/UNIQUE_SETUPS.md#using-xrpljs-with-vite-react).
+
+8. Add script to `package.json`
+
+In your `package.json` file, add the following section if it's not there already:
+
+```json
+"scripts": {
+    "dev": "vite"
+}
+```
+
+### 2. Create the Home Page (Displaying Account & Ledger Details)
+
+In this step, we create a home page that displays account and ledger details.
+
+![Home Page Screenshot](img/js-wallet-home.png)
+
+1. Make a new folder named `src` in the root directory of the project.
+
+2. If not already present, create new files in the root folder named `index.html`, `index.js` and `index.css`.
```

Comment:
I'd recommend swapping the order of these two steps because when running through these steps I got confused and put these 3 files into the `src` folder, which doesn't work. If we have them create the files first, then they won't accidentally add them as a sub-folder file.

Reply:
done